modules/authentication/src/handlers/local.ts

```diff
@@ -979,6 +983,10 @@ export class LocalHandlers implements IAuthenticationStrategy {
     if (isNil(verification)) {
       throw new GrpcError(status.UNAUTHENTICATED, 'Verification unsuccessful');
     }
+    if (verification.delta === -1)
```

Comment:
do not provide extra codes. If delta is anything else besides 0, provide a 400 error (wrong params) and a message that the code is not correct